src/Trakx.CryptoCompare.ApiClient.Tests/Integration/Rest/Core/ThrottledHttpClientHandlerTests.cs

```diff
@@ -16,7 +15,7 @@ public async Task WaitsBetweenQueries()
             var throttleDelayMs = 200;
             var queriesCount = 5;
 
-            var configuration = ConfigurationHelper.GetConfigurationFromEnv<CryptoCompareApiConfiguration>()
+            var configuration = EnvConfigurationHelper.GetConfigurationFromEnv<CryptoCompareApiConfiguration>()
```

Comment:
Could we replace with AWS here? Or do you want to do it in another PR?

Reply:
To replace by Aws we need to create the secrets in AWS for CiCd, not sure they are there

Reply:
but yeah lets make it in a different PR to unlock Market-data